### ISSUE TITLE
'Process' object has no attribute "memory_info'

### ISSUE THREAD

**Issue by bc31164b-cfd5-4a63-8144-875100622b2d** [cptpcrd/pypsutil#2]
Can't get this to run on OSX, and cannot figure what i'm doing wrong, as this is how it works in psutil and it seems how it's covered in the docs too.

**Comment by bc31164b-cfd5-4a63-8144-875100622b2d** [comment]
ok so reading the tests it seems virtual_memory and swap_memory is missing, so supposedly it's not supported on OSX.

**Comment by cptpcrd** [comment]
> Can't get this to run on OSX, and cannot figure what i'm doing wrong, as this is how it works in psutil and it seems how it's covered in the docs too.

Many of `pypsutil`'s features, including `Process.memory_info()`, were written after I released v0.1.0 on PyPI. As a result, the docs (which are built off of `master`) describe some features that aren't available if you just install the package from PyPI. Here are the [docs for v0.1.0](https://pypsutil.readthedocs.io/en/v0.1.0/), so you can see what's supported there. I meant to make a new release a while ago but never got around to it.

Sometime soon, I'll take a look and see if there's anything else I want to get in, then perhaps make a v0.2.0 release. In the meantime, you should be able to install `pypsutil` from the `master` branch of the git repo with one of the following (depending on your situation):

If you just want to install it:
```bash
pip install -e 'git+https://github.com/cptpcrd/pypsutil.git@master#egg=pypsutil'
```

If you use `virtualenv`, add this to your `requirements.txt`:
```
-e git+https://github.com/cptpcrd/pypsutil.git@master#egg=pypsutil
```

If you use `pipenv`:
```bash
pipenv install -e 'git+https://github.com/cptpcrd/pypsutil.git@master#egg=pypsutil'
```

I try to keep `master` in a usable state, so don't worry too much about breakage.

**Comment by bc31164b-cfd5-4a63-8144-875100622b2d** [comment]
Excellent thank you.

**Comment by cptpcrd** [comment]
Released v0.2.0:
https://github.com/cptpcrd/pypsutil/releases/tag/v0.2.0
https://pypi.org/project/pypsutil/#history
